extras/cloudshell/tutorial.md

```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
 
 Let's get started!
 
-## Kubernetes cluster setup
+## Select a GCP project
 
-Set up a Kubernetes cluster using the instructions below for either **minikube** or **GKE**.
+In order to create a Google Kubernetes Engine cluster, you'll need to create a Google Cloud Platform project or use an existing project.
 
----
-### Minikube
+Note: If you choose to deploy to minikube instead of GKE, you can skip this step.
 
-Minikube creates a local Kubernetes cluster on Cloud Shell.
+<walkthrough-project-setup></walkthrough-project-setup>
 
-1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+## Kubernetes cluster setup
 
-2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+This section shows you how to set up a Kubernetes cluster using either **minikube** or **GKE**.
 
-3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+1. Navigate to the <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer">**Clusters**</walkthrough-editor-spotlight> explorer in the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes sidebar</walkthrough-editor-spotlight> using the left side Activity bar.
 
-Once minikube has finished starting, you're ready to move on to the next step.
+2. Click <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer-add-cluster">'+'</walkthrough-editor-spotlight> in the title bar of the Clusters explorer to create a new cluster.
 
+3. Choose either **Minikube** or **Google Kubernetes Engine** and follow the instructions below.
 
---- 
 
-### GKE
+### Minikube
 
-In order to create a GKE cluster, you'll need to **[create a Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)** or use an existing project.
+4. Select the **minikube** profile.
 
-1. Access the command palette by going to **View > Find Command**.
+5. Select **Start**. Cloud Code will initiate a minikube cluster.
+
+6. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+
+
+### GKE
 
-2. Run the command **"Cloud Code: Create GKE cluster"**.
+4. Click **+ Create a new GKE Cluster**.
```

Comment:
Before this step, there should be a "select the google kubernetes engine profile" bullet point to be consistent with the minikube section

Reply:
Just noticing that there's already a common step that says "pick either minikube or gke", in which case, I'd either remove the common step, _or_ I'd remove the "select the minikube profile" altogether, since if we keep both the step would be repeated.

Reply:
Ah, there is a separate step to select the profile for minikube, but there isn't one for GKE.

Profiles are minikube's way of naming and managing clusters, and selecting a minikube profile is equivalent to selecting a GKE cluster. 

And Cloud Shell only has one minikube profile by default - "minikube"! So it looks like a redundant step in the instructions. I've added a link to the minikube docs about profiles to make it more clear that this is a unique step, do you think this is sufficient?

Reply:
Okay that makes sense!!